crt_portal/cts_forms/templates/forms/report_base.html

```diff
@@ -95,12 +95,11 @@ <h2 class="margin-0" tabindex="0">{{ current_step_title }}</h2>
                    value="{% trans 'Next' %}"
                    class="usa-button" />
             {% if wizard.steps.prev %}
-              <br/>
               <button name="wizard_goto_step"
                       type="submit"
                       label="previous step"
                       value="{{ wizard.steps.prev }}"
-                      class="look-like-link margin-top-2">
+                      class="look-like-link margin-top-2 back-button">
```

Comment:
Here, lets use the `display-block` helper class that USWDS exposes.

Reply:
Good idea, thanks!